src/DensityChart/DensityChart.js

```diff
@@ -73,11 +74,24 @@ export default class DensityChart extends PureComponent {
     }
 
     componentDidUpdate(prevProps) {
-        const { brushDomainMin, brushDomainMax, densityChartXScale } = this.props;
+        let min = this.props.brushDomainMin;
+        let max = this.props.brushDomainMax;
+
+        const { densityChartXScale } = this.props;
+
+        if (max >= this.props.timeDomainMax) {
```

Comment:
What is this for?

Reply:
This is used in order to  not be possible drag a brush area to values  outside of the `timeDomain` imposed by the data

Reply:
But how could that happen? The maximum value of the density chart is the maximum of the data

Reply:
did you have seen the gif at #26 ?